src/main.rs

```diff
@@ -23,31 +23,35 @@
 #![feature(once_cell)]
 #![feature(string_remove_matches)]
 #![feature(exclusive_range_pattern)]
+#![feature(if_let_guard)]
 
 extern crate core;
 
 pub mod assembler;
 pub mod cli;
 pub mod file_utils;
-pub mod formatter;
-pub mod parser;
+pub mod parser2;
```

Comment:
Why "parser2" instead of just "parser"?

Reply:
the original parser was called parser and I didn't remove it at first, that's why 😂 will change